routers/api/v1/api.go

```diff
@@ -119,6 +120,9 @@ func sudo() func(ctx *context.APIContext) {
 					}
 					return
 				}
+
+				audit.Record(audit.UserImpersonation, ctx.Doer, ctx.Doer, ctx.Doer, "User %s impersonating user %s.", ctx.Doer.Name, user.Name)
```

Comment:
Sounds wrong that a user impersonates themselves.
Do you mean
```suggestion
				audit.Record(audit.UserImpersonation, ctx.Doer, ctx.Doer, user, "User %s impersonating user %s.", ctx.Doer.Name, user.Name)
```
?